app.js

```diff
@@ -6,7 +6,106 @@
 function loadPage() {}
 
 /* Components */
+const playerChoiceDisplay = document.getElementById('player-choice');
+const computerChoiceDisplay = document.getElementById('computer-choice');
+const resultDisplay = document.getElementById('result');
 
+const possibleChoices = document.querySelectorAll('button');
+
+const winsDisplay = document.getElementById('wins-display');
+const lossesDisplay = document.getElementById('losses-display');
+const drawsDisplay = document.getElementById('draws-display');
+//const totalDisplay = document.getElementById('games-total');
+
+function displayScoreboard() {
+    winsDisplay.textContent = userWins;
+    lossesDisplay.textContent = userLosses;
+    drawsDisplay.textContent = draws;
+    //totalDisplay.textContent = total;
+}
+
+let userChoice;
+let computerChoice;
+let result;
+
+let userWins = 0;
+let userLosses = 0;
+let draws = 0;
+//let total = draws + userWins + userLosses;
+
+possibleChoices.forEach((possibleChoice) =>
+    possibleChoice.addEventListener('click', (e) => {
+        userChoice = e.target.id;
+        playerChoiceDisplay.innerHTML = userChoice;
+        generateComputerChoice();
+        getResult();
+    })
+);
+
+function generateComputerChoice() {
+    const randomNumber = Math.floor(Math.random() * possibleChoices.length) + 1;
+
+    if (randomNumber === 1) {
+        computerChoice = 'Rock';
+    }
+    if (randomNumber === 2) {
+        computerChoice = 'Paper';
+    }
+    if (randomNumber === 3) {
+        computerChoice = 'Scissors';
+    }
+    computerChoiceDisplay.innerHTML = computerChoice;
+}
+
+function getResult() {
+    if (computerChoice === userChoice) {
+        result = 'Draw!';
+        displayScoreboard();
+        draws++;
```

Comment:
The draws++ (and all other incrementers) needs to move above displayScoreboard. That is why your counter is off by one right now.